Client/game_sa/CFxSA.h

```diff
@@ -33,6 +33,7 @@
 #define FUNC_CFx_TriggerWaterSplash        0x4a1070
 #define FUNC_CFx_TriggerBulletSplash       0x4a10e0
 #define FUNC_CFx_TriggerFootSplash         0x4a1150
+#define SHADOW_BASE_TEXTURE_OFFSET         0xC403E0
```

Comment:
Since this value is only used at runtime, you should use a constant expression. Something like this:

```suggestion
constexpr auto SHADOW_BASE_TEXTURE_OFFSET = 0xC403E0;
```

But I don't know if using auto here will resolve to a type that we're happy with.